components/service/glean/README.md

```diff
@@ -95,6 +95,18 @@ object parameter of the `Glean.initialize` method. For example:
 Glean.initialize(applicationContext, Configuration(channel = "beta"))
 ```
 
+### Clearing user-lifetime metrics
+
+"User-lifetime metrics" are metrics that are persistent for the lifetime of the
+user profile. Your app may need to reset these metrics when the profile is
+cleared, or, depending on policy, when telemetry is disabled.
+
+To clear user-lifetime metrics, call:
```

Comment:
I think this requires a bit of design or discussion: this is a product-level feature that *might* happen automatically when turning telemetry off, without requiring any additional call from users and thus reducing our API surface.

You might want to check in with @georgf about this.

Reply:
What are the use-cases when we would expect this to happen?

I would like exposing the resetting of user lifetime metrics outside of Glean, this is easy to use incorrectly.
I'd rather see us expose functions that have a clear purpose and intent.